lib/formatters/getMatchingOrgs.js

```diff
@@ -0,0 +1,26 @@
+'use strict';
+
+function byTagsLength (a, b) {
+  return a.tags.length < b.tags.length;
+}
+
+module.exports = function (rows) {
+  var orgs = rows.reduce(function (obj, row) {
+    // if this is a new org
+    if (!obj[row.or_id]) {
```

Comment:
or_id was chosen because there was already an org_id.. i have cleaned it up now so that we only have or_id now and it still works fine.